frontend/packages/helm-plugin/src/actions/types.ts

```diff
@@ -1,6 +1,7 @@
 export type HelmActionsScope = {
   releaseName: string;
   namespace: string;
+  revisions?: string | number;
```

Comment:
I think now it makes sense to just send the `HelmRelease` object as context. The provider can pull whatever data it needs out of it.